ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/execution/ExpectedWithdrawals.java

```diff
@@ -289,6 +289,37 @@ public void processWithdrawals(
         genericState, schemaDefinitionsCapella, beaconStateMutators, specConfigCapella);
   }
 
+  /*
+   * <spec function="process_withdrawals" fork="electra">
+   * def process_withdrawals(state: BeaconState, payload: ExecutionPayload) -> None:
+   *     # [Modified in Electra:EIP7251]
+   *     expected_withdrawals, processed_partial_withdrawals_count = get_expected_withdrawals(state)
+   *
```

Comment:
do we want the whole function? i can see why the function name is useful and potentially a URL, but maybe the whole spec fn is too much? would be interested in other opinions here for sure.

Reply:
Yeah it certainly can be a lot, but that's what makes it valuable.

* If the spec changes, the CI action will notify devs. This isn't an option without the whole thing.
* It makes it easier for auditors to compare implementation to specification.
* When implementing new spec, having the whole spec there will make implementation easier.
* The spec tags themselves makes it easier to location important things.

How do you feel about a "diff" option, something like this:

```diff
<spec function="process_withdrawals" fork="electra" style="diff">
2c2,4
<     expected_withdrawals = get_expected_withdrawals(state)
---
>     # [Modified in Electra:EIP7251]
>     expected_withdrawals, processed_partial_withdrawals_count = get_expected_withdrawals(state)
>
7a10,12
>     # Update pending partial withdrawals [New in Electra:EIP7251]
>     state.pending_partial_withdrawals = state.pending_partial_withdrawals[processed_partial_withdrawals_count:]
>
</spec>
```

Reply:
I added a `diff` style option which might help here.

Please see: https://github.com/jtraglia/eth-spec-tags/blob/main/README.md#style